args.zig

```diff
@@ -475,7 +475,7 @@ fn convertArgumentValue(comptime T: type, allocator: std.mem.Allocator, textInpu
                 // This is equivalent to allocator.dupeZ but works with any sentinel.
                 if (comptime std.meta.sentinel(T)) |sentinel| {
                     const data = try allocator.alloc(u8, textInput.len + 1);
-                    std.mem.copy(u8, data, textInput);
+                    @memcpy(data, textInput);
```

Comment:
Will assert that `data.len == textInput.len` which is not given. You should slice data here